bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/TextLayout.java

```diff
@@ -1862,8 +1864,8 @@ Rectangle getBoundsInPixels (int start, int end) {
 		}
 		left = Math.min(left, runLead);
```

Comment:
that looks weird, left and right was not scaled up before, but top and bottom was. Is everything in pixels now? but then the call in 1789 is weird to not be scaled down

Reply:
No its like that because this method is called from a public method and we dont have a GC there so, these are some places where we still have to work. Remember, TextLayout is incomplete at the moment. We will need to talk about it.